tools/hyperref-check.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -x
+
+# Extracting all broken hyperref detected by PDFTex
+for file in "./main/acle.md" "./morello/morello.md" "./mve_intrinsics/mve.md" "./neon_intrinsics/advsimd.md"; do
+  echo "Checking $file..."
+
+  broken_refs=`pandoc $file --verbose --fail-if-warnings -o pdfs/tmp.pdf 2>&1 | \
```

Comment:
it is the output of pandoc that should go to a file:

```
pandoc $file --verbose --fail-if-warnings -o pdfs/tmp.pdf >& erroutput.txt
cat erroutput.txt | grep -E 'pdfTeX warning \(dest\): name{[^}]+}' | sed -E 's/.*name\{([^}]+)\}.*/\1/' | sort | uniq > list_of_broken_links
number_of_broken_refs=`cat list_of_broken_links " wc -l`

if [[ "$number_pf_broken_refs" -gt 0 ]]
    echo (the warning)
    cat list_of_broken_links
    exit 1
fi

```

Reply:
I'm confused as to the syntax of this - it's not outputting anything on my local end.